packages/nameguard-sdk/src/index.ts

```diff
@@ -559,25 +578,37 @@ export class NameGuard {
    *
    *
    * @param {string[]} names The list of names for NameGuard to inspect.
-   * @param {InspectNameOptions} options The options for the inspection.
    * @returns {Promise<BulkConsolidatedNameGuardReport>} A promise that resolves with a list of `ConsolidatedNameGuardReport` values for each name queried in the bulk inspection. Check the `inspected` field of each report to determine if the name was fully inspected or inspected in a limited way for performance reasons.
    */
```

Comment:
Let's add a `@throws ...` clause to the documentation for each of the relevant functions that communicates the idea that these functions throw a `NameGuardError` if anything goes wrong.

Reply:
Good idea @lightwalker-eth, I've added this, as well as some `@example` doc blocks.